web/js/containers/sidebar/smart-handoff.js

```diff
@@ -229,16 +229,47 @@ class SmartHandoff extends Component {
     }
   }
 
+  initalBoundary(boundaries, setExtent) {
+    const { proj, map, selectedCollection } = this.props;
+
+    const lonlats = imageUtilGetCoordsFromPixelValues(
```

Comment:
A lot of this code looks like the same as what we have in `onBoundaryChange()`.  Do you think you could find a way to refactor this a little so it can be re-used?